### DISCUSSION TITLE
Tokio runtime error

### DISCUSSION THREAD

**Discussion by hariknine** [tokio-rs/tokio#3057] [Q&A] [answered]
```
thread 'main' panicked at 'not currently running on the Tokio runtime.
```
I'm new to rust and tried to use reqwest (with async) .This is the error I'm getting. What am i doing wrong?

**Comment by carllerche** [comment, marked as answer]
You are probably mixing tokio versions. `reqwest` uses Tokio 0.2 still. You can stick w/ Tokio 0.2 until reqest updates or try out the compat shim: https://github.com/LucioFranco/tokio-compat-02/

**Reply by hariknine** [reply]
That's probably it. i tried it with 0.3.1. Ill surely try with 0.2

**Reply by hariknine** [reply]
It's working now thank you for the quick reply.
